### PR TITLE
Prevent gnome race cond in ensure_unlocked_desktop

### PR DESCRIPTION
**Pull request by asdil12**
Make sure that the VM is responsive.

See also: https://progress.opensuse.org/issues/14064

Tested with opensuse-minimalx and sles-gnome.

### CODE REVIEW

File: tests/console/consoletest_finish.pm [hunk 1]
```diff
@@ -8,8 +8,8 @@
 # notice and this notice are preserved.  This file is offered as-is,
 # without any warranty.
 
-# G-Summary: Rework the tests layout.
-# G-Maintainer: Alberto Planas <aplanas@suse.com>
+# Summary: Cleanup and switch (back) to X11
+# Maintainer: Alberto Planas <aplanas@suse.com>
```
Comment (okurz) [inline tests/console/consoletest_finish.pm]: Please update to someone recently working on this module. I don't mind if you add me, i.e. "okurz@suse.de"

File: lib/utils.pm [hunk 2]
```diff
@@ -361,35 +361,58 @@ sub workaround_type_encrypted_passphrase {
...
+        push @$tags, 'desktop-runner';
+    }
+    else {
+        push @$tags, 'generic-desktop';
+    }
+
+    # wait for run prompt or screen locker
+    push @$tags, @$extra_tags;
     push @$tags, 'screenlock';
+    push @$tags, 'gnome-screenlock-password';
```
Comment (okurz) [inline lib/utils.pm]: that's gnome specific, shouldn't it be used up to line 383?

**Comment by lnussel** [comment]
Looks like this breaks valid uses cases that just take too long:
https://openqa.opensuse.org/tests/281486#step/consoletest_finish/9

**Comment by coolo** [comment]
it worked pretty fine for 9 days though :)

**Comment by asdil12** [comment]
I guess here we ran into the old framebuffer issue, again. The VM/worker is just more busy than I expected. The clock didn't change since consoletest_setup.

**Comment by coolo** [comment]
I just verified with @asdil12 that the link provided by @lnussel is indeed a correctly failed test. The result  could be a bit easier to review though, so I asked @asdil12 to refactor one more time :)

**Pull request merged**